src/pages/Landing/LandingPage.tsx

```diff
@@ -43,6 +43,10 @@ export function LandingPage() {
     organization.name.toLowerCase().includes(searchQuery.toLowerCase()),
   );
 
+  const orgType = t(
+    `SYSTEM__govt_org_type__${organizations[0]?.metadata?.govt_org_type || "district"}`,
+  ).toLowerCase();
```

Comment:
we can't hardcode any fallbacks such as "district"/"state" etc. in our codebase; these terms may not exist in all territories;